lighthouse-core/lib/asset-saver.js

```diff
@@ -33,9 +33,9 @@ const devtoolsLogSuffix = '.devtoolslog.json';
  * Load artifacts object from files located within basePath
  * Also save the traces to their own files
  * @param {string} basePath
- * @return {Promise<LH.Artifacts>}
+ * @return {LH.Artifacts}
  */
-async function loadArtifacts(basePath) {
+function loadArtifacts(basePath) {
```

Comment:
fix to keep smokehouse's `runLighthouse` sync. Since we dropped the streaming JSON loader in #6099, this doesn't need to be async anyways

Reply:
`runLighthouse` needs to become `async` for #8962 (which still needs review btw :P)

Reply:
> runLighthouse needs to become async for #8962 (which still needs review btw :P)

ha, fair. Nothing is async in it anyways, though, so saves us a microtask or two :)